demo/campaign/src/common/blocks/CommonImageBlock.tsx

```diff
@@ -0,0 +1,74 @@
+import { calculateInheritAspectRatio, generateImageUrl } from "@comet/cms-site";
+import { MjmlImage, MjmlStyle } from "@luma-team/mjml-react";
+import { PixelImageBlockData } from "@src/blocks.generated";
+import { css } from "@src/util/stylesHelper";
+import { theme } from "@src/util/theme";
+import * as React from "react";
+
+import { getDamAllowedImageWidth } from "../helpers/imageBlockHelpers";
+
+interface Props extends React.ComponentProps<typeof MjmlImage> {
+    data: PixelImageBlockData;
+    desktopRenderWidth: number;
+    growToOverflowWidth?: boolean;
+}
+
+export const commonImageBlockStyles = (
+    <MjmlStyle>{css`
+        @media (max-width: ${theme.mailSize.contentWidth - 1}px) {
+            .image-block > table > tbody > tr > td {
+                width: 100% !important;
+            }
+
+            .image-block img {
+                height: auto !important;
+            }
+        }
```

Comment:
The responsive behavior doesn't work correctly. 
Please make sure the aspect ratio is always used correctly and that the image is scaled down, as soon, as the width of the window is lower than the width of the mail body. 

https://github.com/user-attachments/assets/1d2e242e-ec16-44a2-8f10-80f9db43036b

Reply:
The incorrect behaviour was caused by missing styling of the `IndentedSectionGroup`. 
Styling was fixed in this commit: https://github.com/vivid-planet/comet-brevo-module/pull/57/commits/712bd92a60af1863c8204bf41014e2f3b01e5be7

Reply:
The aspect ratio still changes, when the responsive styling is applied.